lib/wings/hydra/pcdm/models/concerns/pcdm_valkyrie_behavior.rb

```diff
@@ -30,27 +38,74 @@ def parent_collection_ids(valkyrie: false)
         return member_of_collection_ids if valkyrie
         member_of_collection_ids.map(&:id)
       end
+      # alias member_of_collection_ids child_object_ids # TODO: Cannot alias in this way because member_of_collection_ids method is already defined thru the attribute definition.
 
       ##
       # Gives the subset of #members that are PCDM objects
       # @return [Enumerable<ActiveFedora::Base> | Enumerable<Valkyrie::Resource>] an enumerable over the members
       #   that are PCDM objects
+      # NOTE: A collection could be added to members, but for Hyrax collections are stored using member_of_collection_ids.
+      #       As long as this rule holds, then members and ordered_members are equivalent to objects/child_objects.
+      # TODO: Do we want to support members separate from child_objects (see NOTE)
       def child_objects(valkyrie: false)
         af_objects = Wings::ActiveFedoraConverter.new(resource: self).convert.objects
         return af_objects unless valkyrie
         af_objects.map(&:valkyrie_resource)
       end
       alias objects child_objects
+      alias ordered_objects child_objects
       alias members child_objects
       alias ordered_members child_objects
 
       ##
       # Gives a subset of #member_ids, where all elements are PCDM objects.
       # @return [Enumerable<String> | Enumerable<Valkyrie::ID] the object ids
+      # NOTE: A collection could be added to member_ids, but for Hyrax collections are stored using member_of_collection_ids.
+      #       As long as this rule holds, then member_ids and ordered_member_ids are equivalent to object_ids/child_object_ids.
+      # TODO: Do we want to support member_ids separate from child_objects (see NOTE)
       def child_object_ids(valkyrie: false)
         child_objects(valkyrie: valkyrie).map(&:id)
       end
       alias object_ids child_object_ids
+      alias ordered_object_ids child_object_ids
+      # alias member_ids child_object_ids # TODO: Cannot alias in this way because member_ids method is already defined thru the attribute definition.
+      alias ordered_members_ids child_object_ids
```

Comment:
The #child_object_ids method returns the ids of member objects in order, so #ordered_member_ids is aliased to #child_object_ids. This addresses Issue #3583.